Contest_API.md

```diff
@@ -305,22 +300,36 @@ with JSON data
 }
 ```
 
-### Extensibility
-
-This specification is meant to cover the basic data of contests, with
-the idea that server/client implementations can extend this with more
-data and/or roles. In particular, this specification already lists some
-endpoints or specific properties as optional. The following guidelines
-are meant to ease extensibility.
-
-  - Clients should accept extra properties in endpoints, that are not
-    specified here.
-  - Servers should not expect clients to recognize more than the basic,
-    required specification.
-  - In this specification and extensions, a property with value `null`
-    may be left out by the server (i.e. not be present). A client must
-    treat a property with value `null` equivalently as that property
-    not being present.
+### Capabilities
+
+The API specifies several
+capabilities that define behaviours that clients can expect and
+actions they can perform. For instance, a team account will typically
+have access to a "team_submit" capability that allows a team to perform
+POST operations on the submissions endpoint, but doesn't allow it to
+set the submission id or timestamp; an administrator may have access
+to a "contest_start" capability that allows it to PATCH the start
+time of the contest. These coarse-grained capabilities allow more
+flexibility for contest administrators and tools to define capabilities
+that match the requirements of a specific contest - e.g. whether teams
```

Comment:
This is really nitpicky, but I think a comma is better here, or at least more common. Also if used, should this be an em-dash (or an en-dash)?
```suggestion
that match the requirements of a specific contest, e.g. whether teams
```